src/style/font/font_desc.rs

```diff
@@ -45,20 +45,63 @@ impl FontTransform {
 #[derive(Clone)]
 pub struct FontDesc<'a> {
     size: f64,
-    name: &'a str,
+    family: FontFamily<'a>,
     data: FontResult<FontDataInternal>,
     transform: FontTransform,
 }
 
+/// Describes font family
+#[derive(Clone, Copy)]
+pub enum FontFamily<'a> {
+    Sans,
```

Comment:
Isn't "sans" same as "sans-serif" but "serif" is different ?
And for CSS it has "serif", "sans-serif" and "monospace". So it seems we should have `Serif` other than `Sans` https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#%3Cgeneric-name%3E   

I am not familiar with fonts. But based on what I learned it seems to be this. I may be wrong on this.

Reply:
Ah, right. I'm constantly misuse them ;)